dspace-spring-rest/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -306,9 +330,215 @@ public void findOneTestWrongUUID() throws Exception {
         Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
         Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
 
-        getClient().perform(get("/api/core/items/" + UUID.randomUUID()))
-                   .andExpect(status().isNotFound());
+        String token = getAuthToken(admin.getEmail(), password);
+
+        getClient(token).perform(get("/api/core/items/" + UUID.randomUUID()))
+                        .andExpect(status().isNotFound());
         ;
     }
 
+    @Test
+    public void embargoAnonymousAccessTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and two collections.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                                          .withName("Parent Community")
+                                          .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                                           .withName("Sub Community")
+                                           .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+
+        //2. Three public items that are readable by Anonymous with different subjects
+        Item embargoedItem1 = ItemBuilder.createItem(context, col1)
+                                         .withTitle("embargoed item 1")
+                                         .withIssueDate("2017-10-17")
+                                         .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                                         .withSubject("ExtraEntry")
+                                         .withEmbargoPeriod("6 months")
+                                         .build();
+
+        //2. Three public items that are readable by Anonymous with different subjects
+        Item publicItem1 = ItemBuilder.createItem(context, col1)
+                                      .withTitle("Public item 1")
+                                      .withIssueDate("2017-10-17")
+                                      .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                                      .withSubject("ExtraEntry")
+                                      .build();
+
+        context.restoreAuthSystemState();
+
+
+        getClient().perform(get("/api/core/items/" + embargoedItem1.getID()))
```

Comment:
Since publicItem1 was created, should a retrieve test be performed here to illustrate the successful return?